.github/workflows/test.yml

```diff
@@ -20,7 +20,12 @@ jobs:
     outputs:
       should_skip: ${{ steps.skip_check.outputs.should_skip }}
       should_skip_ghcide: ${{ steps.skip_ghcide_check.outputs.should_skip }}
+      ghcs: ${{ steps.ghcs.outputs.ghcs }}
     steps:
+      # Need the repo checked out in order to read the file
+      - uses: actions/checkout@v3
+      - id: ghcs
+        run: echo "ghcs=$(cat ./.github/workflows/supported-ghc-versions.json)" >> $GITHUB_OUTPUT
```

Comment:
Can we maybe extract these steps to https://github.com/haskell/haskell-language-server/blob/mpj/dynamic-matrices/.github/actions/setup-build/action.yml if everybody needs it?

Reply:
Not quite, I think, because it has to be in a separate job. But we could make another reusable workflow for it.

Reply:
Although tbh the reusable part is essentially this one line, so I'm not sure if it's worth it :thinking: